test/relative-time.js

```diff
@@ -482,6 +482,24 @@ suite('relative-time', function () {
       assert.equal(time.shadowRoot.textContent, '4 months ago')
     })
 
+    test('rewrites from last few days of month to smaller last month', async () => {
+      freezeTime(new Date(2024, 4, 31))
+      const time = document.createElement('relative-time')
+      time.setAttribute('tense', 'past')
+      time.setAttribute('datetime', '2024-04-30T00:00:00Z')
+      await Promise.resolve()
+      assert.equal(time.shadowRoot.textContent, 'last month')
+    })
+
+    test('rewrites from last few days of month to smaller previous month', async () => {
+      freezeTime(new Date(2024, 4, 31))
+      const time = document.createElement('relative-time')
+      time.setAttribute('tense', 'past')
+      time.setAttribute('datetime', '2024-02-29T00:00:00Z')
+      await Promise.resolve()
+      assert.equal(time.shadowRoot.textContent, '3 months ago')
```

Comment:
This one doesn't make much sense to me. This seems to be saying that someone on the 31st March would say the 29th Feb was 3 months ago?